docs/source/parse_input.rst

```diff
@@ -0,0 +1,67 @@
+Parse input
+===========
+
+One of the flask-ninja features is the automatic parsing of path and query arguments and also request bodies and providing them via the arguments of the function.
```

Comment:
unify the naming please

Reply:
and maybe let's make this a list/bullet points instead of this long sentence please